zk/hermez_db/db.go

```diff
@@ -46,7 +46,7 @@ const L1_INFO_LEAVES = "l1_info_leaves"                                 // l1 in
 const L1_INFO_ROOTS = "l1_info_roots"                                   // root hash -> l1 info tree index
 const INVALID_BATCHES = "invalid_batches"                               // batch number -> true
 const BATCH_PARTIALLY_PROCESSED = "batch_partially_processed"           // batch number -> true
-const LOCAL_EXIT_ROOTS = "local_exit_roots"                             // l2 block number -> local exit root
+const LOCAL_EXIT_ROOTS = "local_exit_roots"                             // batch number -> local exit root
```

Comment:
I think with us moving to block level storing of this kind of information it should stay as block -> LER

Reply:
yeah - it was just a wrong comment, maybe I should adjust this completely.